src/parser/esprima_cpp/esprima.cpp

```diff
@@ -5854,14 +5819,11 @@ class Parser {
 
         MetaNode endNode = this->createNode();
         this->currentScopeContext->m_bodyEndLOC.index = endNode.index;
-#if !(defined NDEBUG) || defined ESCARGOT_DEBUGGER
+#if !(defined NDEBUG)
         this->currentScopeContext->m_bodyEndLOC.line = endNode.line;
         this->currentScopeContext->m_bodyEndLOC.column = endNode.column;
 #endif
```

Comment:
If debugger does not use the ExtendedNodeLOC for `m_bodyEndLOC`, then other part related to it should be fixed too.
The below is an example.
https://github.com/Samsung/escargot/blob/b977ab7e54a10fdeac7d618e0a90308f7d723709/src/parser/ast/ASTContext.h#L215-L217